modules/distr-tracing-config-otel-collector.adoc

```diff
@@ -238,6 +238,44 @@ Zipkin receiver ingests data in the Zipkin V1 and V2 formats.
 <1> The Zipkin HTTP endpoint. If omitted the default `+0.0.0.0:9411+` is used.
 <2> The TLS server side configuration. See the OTLP receiver configuration section for more details.
 
+
+=== Processors
+
+==== Resource Detection Processor.
+
+Resource Detection processor is designed to identify host resource details in alignment with OpenTelemetry's resource semantic standards.
+It can then either add or replace the resource values in telemetry data using this detected information.
```

Comment:
If we mention the replace here, we should add the `overwrite` field in the example config.

Reply:
I've linked to this comment in our team chat channel.